### ISSUE TITLE
Feature: Please expose keyFor() function

### ISSUE THREAD

**Issue by styxlab** [jaketrent/react-social-icons#92]
Thanks for this nice repo!

I would like to propose exposing the `keyFor()` function. Why? Because it allows to programatically check, whether a social icon is detected.

In my use case, I want save the name of the detected network in the database in addition to showing an icon in the UI. I can imagine there are many other use cases that would benefit from that function.

**Comment by couetilc** [comment]
Thank you @styxlab, released in v5.5.0 https://github.com/jaketrent/react-social-icons/releases/tag/v5.5.0

**Comment by styxlab** [comment]
@couetilc: Thanks so much for providing this feature. However, I am using this ins typescript project and the import cannot see the exposed function.

```
import { keyFor } from 'react-social-icons'
```

is giving me a compile error:

```
Type error: Module '"react-social-icons"' has no exported member 'keyFor'.
```

I think it's due to missing type definitions, but am not sure how to include those in this project. Could you please have a second look at this?

**Comment by styxlab** [comment]
For anyone coming across the missing type issue, here is the workaround I am currently using. Just add the following type definition in your `types.d.ts` file of your project directory.

```
declare module 'react-social-icons' {
  export function keyFor(value: string): string
}
```
After that, the newly exposed `keyFor` function is recognized by your typescript compiler.

I still hope, the types can be fixed within the main package, so this workaround is not needed anymore.

**Comment by couetilc** [comment]
@styxlab addressed in v5.5.1 https://github.com/jaketrent/react-social-icons/releases/tag/v5.5.1
